components/support/rc_crypto/src/hkdf.rs

```diff
@@ -98,7 +94,7 @@ mod tests {
     #[test]
     fn hkdf_rejects_gigantic_output_buffers() {
         let salt = hmac::SigningKey::new(&digest::SHA256, b"salt");
-        let mut out = vec![0u8; NSS_MAX_DERIVED_KEY_MATERIAL + 1];
+        let mut out = vec![0u8; 8160 + 1]; // RFC maximum (hashlen * 255) + 1
```

Comment:
@rfk heads up https://bugzilla.mozilla.org/show_bug.cgi?id=1553372 was resolved and failed that test :)